pkg/querier/queryrange/query_range.go

```diff
@@ -31,8 +31,12 @@ import (
 const StatusSuccess = "success"
 
 var (
-	matrix            = model.ValMatrix.String()
-	json              = jsoniter.ConfigCompatibleWithStandardLibrary
+	matrix = model.ValMatrix.String()
+	json   = jsoniter.Config{
+		EscapeHTML:             false, // No HTML in our responses.
+		SortMapKeys:            true,
```

Comment:
Why is this required?

Reply:
I don't think it's required, but I wasn't able to figure out if we may introduce any regression disabling it (the default config has it enabled). Any thoughts?

Reply:
I expect some clients are relying on it whether they realise it or not. 
I.e. the ordering of series here maps directly to what you see in your dashboard, in at least some cases.

However we should try disabling it (in a separate change) and see if anyone notices.

Reply:
Opened an issue to follow up about it:
https://github.com/cortexproject/cortex/issues/3997